1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,20 @@
-function setAlarm() {}
+function setAlarm() {
+  let timeRemaining = document.getElementById("timeRemaining");
+  let input = document.getElementById("alarmSet");
+  let inputValue = input.value;
+  countdown = setInterval(() => {
+    let minutes = Math.floor(inputValue / 60);  
```

Comment:
Magic number, which you use twice. Lets me the number: `60` a variable with a clear name: `const SECONDS_IN_A_MINUTE=60` for example